WRITING-DOCUMENTATION.md

```diff
@@ -7,12 +7,21 @@ as you write your documentation.
 
 The code uses trunk based development where `master` is the `trunk` branch.
 
-A folder in [app/docs](app/docs) exists for each minor version of Kuma. 
-There is a special folder for the future non patch version of Kuma which is called [dev](app/docs/dev).
+A single sourced folder in [app/_src](app/_src) is used for each version of Kuma. We use a Jekyll plugin to dynamically generate pages from a single source file.
+
+For the future non-patch versions of Kuma, changes can be made to the [docs_nav_kuma_dev.yml](app/_data/docs_nav_kuma_dev.yml) file. 
 
 ## Writing docs for a new feature
 
-If you are writing docs for a new feature you'll want to add it in the [dev](app/docs/dev) folder.
+If you are writing docs for a new feature you'll want to add it in the [src](app/_src) folder.
+
+Since content is single sourced, you must use [conditional rendering](https://docs.konghq.com/contributing/conditional-rendering/) to ensure that the new feature content only displays for that version. For example:
+
+```
+{% if_version eq:2.1.x %}
```

Comment:
Should this be `gt:2.1.x` ?

Reply:
`eq` is correct. That means the version has to be equal to 2.1.x for the content to display. If we used `gte`, it would display for 2.1.x and later versions.

Reply:
I believe it makes sense to use `gt` as we'd want the new feature also available in future version's docs, we usually don't put docs for just one version (except for behaviour that would be fixed later but that's fairly rare)?

Maybe just enumerating the options (linking to the external docs for syntax is enough) and let the writer pick what's best is what we should do here.